<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>viwe_profile</class>
 <widget class="QWidget" name="viwe_profile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0" colspan="5">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_Account_ID">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(1, 116, 190);</string>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_Account_ID_2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(1, 116, 190);</string>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="pushButton_Skill_2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(1, 116, 190);</string>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_Last_name_2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(1, 116, 190);</string>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_First_name">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(1, 116, 190);</string>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="pushButton_Email">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(1, 116, 190);</string>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="pushButton_Phone_number_2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(1, 116, 190);</string>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="pushButton_Email_2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(1, 116, 190);</string>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="pushButton_Phone_number">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(1, 116, 190);</string>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_First_name_2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(1, 116, 190);</string>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_Last_name">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(1, 116, 190);</string>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QListWidget" name="listWidget_Skill">
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="pushButton_message">
     <property name="text">
      <string>message</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_company_code">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(1, 116, 190);</string>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_company_code_2">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(1, 116, 190);</string>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_company">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(1, 116, 190);</string>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_company_2">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(1, 116, 190);</string>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
